core/src/display_object/movie_clip.rs

```diff
@@ -315,6 +321,75 @@ impl<'gc> MovieClip<'gc> {
         MovieClipWeak(GcCell::downgrade(self.0))
     }
 
+    pub fn new_import_assets(
+        activation: &mut Avm2Activation<'_, 'gc>,
```

Comment:
Not blocking, but since `new_import_assets` doesn't need any state of the Activation, it should take an UpdateContext and construct the Activation itself. Also, it can be a `from_nothing` Activation, it doesn't need to be `from_domain`.

Reply:
I removed the loader info therefore also the activation